### PR TITLE
Added bio/website fields for DSF members.

### PR DESCRIPTION
**Pull request by timgraham**

### CODE REVIEW

File: djangoproject/settings/common.py [hunk 1]
```diff
@@ -28,8 +28,6 @@
 
 CACHE_MIDDLEWARE_KEY_PREFIX = 'django'
 
-CSRF_COOKIE_HTTPONLY = True
-
```
Comment (charettes) [inline djangoproject/settings/common.py]: Is this required by the markdown stuff? If not it should be in another commit at least.

File: djangoproject/urls/www.py [hunk 2]
```diff
@@ -53,6 +53,7 @@
     url(r'^contact/', include('contact.urls')),
     url(r'^foundation/', include('members.urls')),
     url(r'^fundraising/', include('fundraising.urls')),
+    url(r'^markdownx/', include('markdownx.urls')),
```
Comment (charettes) [inline djangoproject/urls/www.py]: I assume this is used by the admin for markdown preview? If it's the case we want to allow access to this view to staff users. Might want to test it's not publicly available as well?

File: djangoproject/settings/common.py (hunk 1 above)
Comment (timgraham) [inline djangoproject/settings/common.py]: Yes, see https://github.com/adi-/django-markdownx/issues/21. Although I thought about doing it in a separate commit anyway. It's a setting that doesn't offer any benefit per https://github.com/django/django/commit/c27104a9c74bc9d9e552d41f53468b103749e110.

Comment (charettes) [inline djangoproject/settings/common.py]: Makes sense. Disabling it in a prior commit explaining why we don't need it wouldn't hurt as you've mentioned.

File: djangoproject/urls/www.py (hunk 2 above)
Comment (timgraham) [inline djangoproject/urls/www.py]: Yes, but all DSF members will be able to edit their bios, probably in a separate view outside the admin. I didn't spot anything in https://github.com/adi-/django-markdownx about security implications of not limiting access to the view. Do you think that's a "service" that someone is going to take advantage of?

Comment (charettes) [inline djangoproject/urls/www.py]: I think this could be a DDoS vector as there's no throttling or validation of user input. Limiting the users allowed to reach this view to DSF members only should be safer. I wonder why `MarkdownxModelAdmin` doesn't take care of adding an `url` (through `get_urls()`) itself instead of suggesting to publicly expose this view.

Comment (MarkusH) [inline djangoproject/urls/www.py]: I agree with @charettes evaluation on DDoS. I rather not have this exposed publicly.

Comment (timgraham) [inline djangoproject/urls/www.py]: As you wish. It's probably better to omit the image upload view anyway.
